klippa/src/graph/layout.rs

```diff
@@ -0,0 +1,197 @@
+//! Read layout tables in a graph
+//!
+use crate::{
+    graph::{Graph, RepackErrorFlags},
+    serialize::{LinkWidth, ObjIdx},
+};
+use fnv::FnvHashMap;
+use write_fonts::types::Scalar;
+struct ExtensionSubtable<'a> {
+    bytes: &'a mut [u8],
+}
+
+impl<'a> ExtensionSubtable<'a> {
+    const FORMAT_BYTE_LEN: usize = 2;
+    const LOOKUP_TYPE_LEN: usize = 2;
+    fn from_graph(graph: &'a mut Graph, obj_idx: ObjIdx) -> Result<Self, RepackErrorFlags> {
+        let ext = graph
+            .obj_data(obj_idx)
+            .map(move |data| Self { bytes: data })
```

Comment:
I don't think `move` is needed here. Only `data` is used, and that is taken as a parameter and not captured from the environment.